utils/fork-tree/src/lib.rs

```diff
@@ -1001,40 +980,22 @@ mod test {
 
 		tree.finalize_root(&"A");
 
-		assert_eq!(
-			tree.best_finalized_number,
-			Some(1),
-		);
+		assert_eq!(tree.best_finalized_number, Some(1),);
 
-		assert_eq!(
-			tree.import("A", 1, (), &is_descendent_of),
-			Err(Error::Revert),
-		);
+		assert_eq!(tree.import("A", 1, (), &is_descendent_of), Err(Error::Revert),);
```

Comment:
I thought the commas are set to be `Vertical` so this last comma should not be there? `,`

Reply:
(aside from the single line additional commas the rest looks parity coding standard to me.